src/test/scala/org/hammerlab/guacamole/HasReferenceRegionSuite.scala

```diff
@@ -0,0 +1,53 @@
+package org.hammerlab.guacamole
+
+import org.scalatest.{Matchers, FunSuite}
+
+
+class HasReferenceRegionSuite extends FunSuite with Matchers {
+
+  test("overlapping reads") {
+    val read1 = TestUtil.makeRead("TCGATCGA", cigar = "8M", mdtag = "8", start = 1L)
+    val read2 = TestUtil.makeRead("TCGATCGA", cigar = "8M", mdtag = "8", start = 5L)
+
+    read1.overlaps(read2) should be(true)
+    read2.overlaps(read1) should be(true)
+
+  }
+
+  test("overlapping reads, different contigs") {
+    val read1 = TestUtil.makeRead("TCGATCGA", cigar = "8M", mdtag = "8", start = 1L)
+    val read2 = TestUtil.makeRead("TCGATCGA", cigar = "8M", mdtag = "8", start = 5L, chr = "chr2")
```

Comment:
@ryan-williams sorry for not looking at this sooner.  I don't think I understand what you mean - isn't the 1-8, 5-13 the test above where they are on the same chromosome?